chingu-voyages/v46-tier2-team-18
# Create Nextjs app, edit scripts

This PR initializes a new Next.js application and updates the package.json to include scripts for ESLint and Prettier. These additions set the foundation for our project and ensure that we maintain code quality and consistency.

**Changes:**
- Initialize a new Next.js app with the default configuration, exception made by using JS instead of TS.
- Update package.json to include the following scripts:

```
"scripts": {
  "dev": "next dev",
  "build": "next build",
  "start": "next start",
  "lint": "next lint",
  "eslint": "eslint .",
  "prettier": "prettier --check ."
}
```

- Add a GitHub Actions workflow for CI/CD that includes steps for running ESLint and Prettier.

**How to Test:**

- Clone this branch locally.
- Run npm install to install all dependencies.
- Run npm run dev to start the development server. The Next.js app should open at http://localhost:3000/.
- Run npm run eslint to run ESLint and check for code quality issues.
- Run npm run prettier to run Prettier and check for formatting issues.

This PR closes #27
